src/plugins/discover/public/embeddable/components/search_embeddable_grid_component.tsx

```diff
@@ -122,6 +136,26 @@ export function SearchEmbeddableGridComponent({
     settings: grid,
   });
 
+  const dataSource = useMemo(() => createDataSource({ dataView, query }), [dataView, query]);
+
+  const timeRange = useMemo(
+    () => (fetchContext ? getTimeRangeFromFetchContext(fetchContext) : undefined),
+    [fetchContext]
+  );
+
+  const cellActionsMetadata = useAdditionalCellActions({
+    dataSource,
+    dataView,
+    query,
+    filters,
+    timeRange,
+  });
+
+  // Security Solution overrides our cell actions -- this is a temporary workaroud to keep
+  // things working as they do currently until we can migrate their actions to One Discover
+  const isInSecuritySolution =
+    useObservable(discoverServices.application.currentAppId$) === 'securitySolutionUI';
```

Comment:
@elastic/security-threat-hunting-investigations I wanted to build this extension point on top of `kbn-cell-actions` since Unified Data Table already supports it, rather than introducing an alternative approach. Unfortunately this conflicts with the saved search embeddable cell action overrides currently active within the Security Solution UI.

So I needed a way to disable the cell actions extension point when it would conflict with the overrides and couldn't figure out another way other than this. I'm open to ideas for a cleaner approach, but my thinking is that this won't be needed eventually anyway since ideally we'll migrate the current Security actions to a context aware Discover profile instead.